scalar/README.md

```
@@ -0,0 +1,140 @@
+	ProxyUrl string
+
+	// Raw Space Url
+	// Optional. Default: doc.json
+	RawSpecUrl string
+}
+```
+
+### Default Config
+```go
+var ConfigDefault = Config{
+	Next:       nil,
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)